<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>troçDeManta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>2</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Torrents</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLineEdit" name="text_busqueda">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="buto_busqueda">
            <property name="text">
             <string>Busca</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboMotors"/>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Opcions</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="check_series">
               <property name="text">
                <string>Serie</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Sense</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Amb</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QFrame" name="frameBaix">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
           </property>
           <item row="0" column="0">
            <widget class="QTreeWidget" name="treeResultat">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <column>
              <property name="text">
               <string>NOM</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>URL</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Informació</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_15">
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Torrent</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="btnDownload">
                <property name="text">
                 <string>Descarrega</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="Line" name="line">
             <property name="cursor">
              <cursorShape>SplitHCursor</cursorShape>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Series</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineFiltra"/>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>LLista</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QRadioButton" name="radioLocal">
                <property name="text">
                 <string>Les meves series</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QRadioButton" name="radioGlobal">
                <property name="text">
                 <string>Online</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="3">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Informació</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="1" column="0">
               <widget class="QFrame" name="frame_5">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_12">
                 <item row="0" column="0">
                  <widget class="QTreeWidget" name="treeEpisodis">
                   <property name="wordWrap">
                    <bool>true</bool>
                   </property>
                   <column>
                    <property name="text">
                     <string>Season</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Episode</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Name</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QFrame" name="frame_4">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_11">
                 <item row="1" column="1">
                  <widget class="QFrame" name="frame_6">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_13">
                    <item row="0" column="0">
                     <widget class="QLabel" name="labelTitol">
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>20</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Titol...</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="labelEpisodi">
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>15</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="labelRating">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QTextBrowser" name="labelSinopsi">
                      <property name="minimumSize">
                       <size>
                        <width>200</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>300</width>
                        <height>16777215</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QFrame" name="frame_7">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_14">
                    <item row="0" column="0">
                     <widget class="QLabel" name="labelImatge">
                      <property name="minimumSize">
                       <size>
                        <width>200</width>
                        <height>300</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>300</width>
                        <height>400</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTreeWidget" name="listShows">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string>TvMaze Id</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Title</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Rating</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="btnDeleteShow">
             <property name="text">
              <string>Borrar</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnAddShow">
             <property name="text">
              <string>Afegir</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>988</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioGlobal</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnAddShow</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>929</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioGlobal</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnDeleteShow</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>970</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioLocal</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnDeleteShow</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioLocal</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnAddShow</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
